src/pages/marketplace/index.js

```diff
@@ -0,0 +1,611 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
whats this copyright for?

Reply:
This will be changed to reflect the same Palo Alto Networks copyright used throughout the production site.